composer.json

```diff
@@ -34,16 +34,16 @@
     },
     "autoload-dev": {
         "psr-4": {
-            "Create_WordPress_Plugin\\Tests\\": "tests"
+            "Alley\\WP\\WP_404_Caching\\Tests\\": "tests"
         }
     },
     "extra": {
         "wordpress-autoloader": {
             "autoload": {
-                "Create_WordPress_Plugin\\": "src"
+                "Alley\\WP\\WP_404_Caching": "src"
             },
             "autoload-dev": {
-                "Create_WordPress_Plugin\\Tests\\": "tests"
+                "Alley\\WP\\WP_404_Caching\\Tests": "tests"
```

Comment:
The placeholders end in `\\` but the new lines don't and it's throwing me off, so I want to highlight that in case it wasn't intentional.

Reply:
Good catch I will fix this!